### PR TITLE
documentation edits

### PR DESCRIPTION
**Pull request by gregcaporaso**
~~Not ready for merge yet - I may have some additional edits as I work through the tutorial.~~

### CODE REVIEW

File: tutorial/tutorial.md [hunk 1]
```diff
@@ -2,117 +2,92 @@
...
-the individual fractions that are separated from a single source and then
-sequenced individually. Thus there are multiple samples per source, and study
-metadata must take this into account.
-
-There are two ways to store such hierarchical metadata: in a single file, or in
-two files, where one represents the source-level metadata and the other the
-sample-level metadata. The qSIP2 plugin accepts both formats.
+The metadata structure of qSIP data is somewhat more complicated than that of more typical 16S sequencing workflows.
+That is because qSIP metadata is hierarchical.
+There are "source" level entities--these are commonly the subjects in your study, and represent a single microbial community, whether enriched with your stable isotope or unenriched.
```
Comment (colinvwood) [inline tutorial/tutorial.md]:
```suggestion
There are "source" level entities--these are commonly the subjects in your study, and represent a single microbial community, whether enriched with the heavy isotope or unenriched.
```

Both the heavy and light isotopes are stable is my understanding.

File: tutorial/tutorial.md [hunk 2]
```diff
@@ -127,26 +102,21 @@ qiime qsip2 create-qsip-data \
...
-Each of the column names has a default value that can be seen by running
-`qiime qsip2 create-qsip-data --help` and reading the resulting command line
-output. If a column in your metadata is already named with this default then
-you do not need to provide that argument. For example, if your
-**source material id** column is alread called `source_mat_id` you can drop
-the `--p-source-mat-id` argument from the command. Note that in this example
-we are dropping the `--p-isotoplog-column` argument from the command because
-our isotoplog column is already named with the default. If you isotopolog
-column has a different name, you should add this argument with that name.
+(Note: When working with your own data, if you have a single metadata file, the software treats this a sample-level metadata, and you would simply not provide the `--p-source-metadata` argument.)
```
Comment (colinvwood) [inline tutorial/tutorial.md]: This is nitpick-y, but I think the single file/double file business is one of the more confusing parts of the workflow. If I were reading through this, I might glaze over anything parenthetical--do you mind if we leave this in the main paragraph?

File: tutorial/tutorial.md [hunk 3]
```diff
@@ -155,33 +125,26 @@ qiime qsip2 plot-weighted-average-densities \
...
-Here, we are interested in seeing if activity differs between our two moisture
-groups, "normal" and "drought".
+The `--p-group` command can be any source-level metadata column that you want to use to facet the resulting plot of weighted average densities (WADs).
+Here, we are interested in seeing if activity differs between our two moisture groups, "normal" and "drought".
 
-To view the visualization you can either go to the QIIME2 visualzation-viewer
-[website](https://view.qiime2.org) and upload your file, or if your computer
-has a screen (i.e. you aren't on a server), run the following on the command
-line.
+To view the visualization you can either [QIIME 2 View](https://view.qiime2.org), or if your computer has a screen (i.e. you aren't on a server), run the following on the command line.
```
Comment (colinvwood) [inline tutorial/tutorial.md]:
```suggestion
To view the visualization you can either use [QIIME 2 View](https://view.qiime2.org), or if your computer has a screen (i.e. you aren't on a server), run the following on the command line.
```

File: tutorial/tutorial.md (hunk 2 above)
Comment (gregcaporaso) [reply tutorial/tutorial.md]: Good thought. If/when we convert this to a Jupyter Book, we can put this in an admonition block to make it stand out. For now, I put the text about running on your own data in one single paragraph, and bold-faced some text to make it stand out: 
Screenshot 2024-09-19 at 5 57 59 AM

How's that?

**Comment by gregcaporaso** [comment]
@colinvwood, I accepted your suggestions and updated the text according to your suggestion. 

I did an interactive rebase locally, so all of the most recent edits are included in the final commit. Since the other commits in this PR are logically different, I think it makes sense this way so you can just rebase and merge (rather than squash and merge).

File: tutorial/tutorial.md (hunk 2 above)
Comment (colinvwood) [reply tutorial/tutorial.md]: Sounds great.

**Pull request merged**